extensions-core/kafka-indexing-service/src/main/java/io/druid/indexing/kafka/KafkaIndexTask.java

```diff
@@ -305,6 +306,12 @@ private static String makeTaskId(String dataSource, int randomBits)
     return Joiner.on("_").join(TYPE, dataSource, suffix);
   }
 
+  @Override
+  public int getPriority()
+  {
+    return getContextValue(Tasks.PRIORITY_KEY, Tasks.DEFAULT_REALTIME_TASK_PRIORITY);
```

Comment:
can we add basic doc about this? I feel like this kind of hidden feature is not good for overall code maintenance and overall ease of use in general.  plus providing a couple of example of when the user wants to use this can be helpful too.

Reply:
There is a [document](https://github.com/druid-io/druid/blob/master/docs/content/ingestion/tasks.md#task-priority). Do you think we need a more detailed one?

Reply:
then this is kind of a bug fix since the feature was here but the default was returning wrong default reading the PR suggests that this is a new thingy.

Reply:
Right. Added a bug label. Thanks.